### ISSUE TITLE
Host helm repo on GitHub Pages

### ISSUE THREAD

**Issue by tmaier** [clockworksoul/helm-elasticsearch#37]
It would be great if there would be helm repo to use for this chart.

You could use TravisCI to package and push it to GitHub Pages.

https://github.com/kubenow/helm-charts is doing this. There you can also find a [.travis.yml](https://github.com/kubenow/helm-charts/blob/master/.travis.yml) file which you could copy. (same OSS license)

**Comment by jschaul** [comment]
Agreed, that would be great. Hosting this repo as a helm chart repository would simplify depending on it from within another chart's `requirements.yaml`. The official docs have an example for how to do this manually via [github pages](https://github.com/kubernetes/helm/blob/master/docs/chart_repository.md#github-pages-example), though of course automating it via travis would be even better.

**Comment by sagikazarmark** [comment]
I already started experimenting with this, I will try to pick this up in the next couple weeks.
